README.md

```diff
@@ -65,8 +65,8 @@ NB:
 
 * run:
 
-        $ docker run -it --rm --name naveed -p8080:8465
-                -v "$(pwd)/config:/app/config" \
+        $ docker run --rm --name naveed -p8080:8465
+                -v "$(pwd)/config:/" \
```

Comment:
that `config` dir seems bogus; it's not actually being used anywhere and doesn't exist by default either - overall the instructions here are confusing; at the very least, assumptions need to be made explicit

Reply:
yup ... this adjustment was just made to keep it consistent with the prior image (as I moved the default config file to `/`)
.
Thinking about this again: this is actually bullshit: it would overwrite `/` :D .. if that's even possible -- sorry